simulator/risc_v/t/unit_test.cpp

```diff
@@ -734,6 +734,25 @@ TEST_CASE("RISCV RV64 gorci")
     }
 }
 
+TEST_CASE("RISCV RV32 shfl")
+{
+    CHECK( RISCVInstr<uint32>(0x091815b3).get_disasm() == "shfl $a1, $a6, $a7");
+    RISCVInstr<uint32> instr("shfl", 0);
+    instr.set_v_src( 0xbf534fdeU, 0);
+    instr.set_v_src( 0x0C, 1);
+    instr.execute();
+    CHECK( instr.get_v_dst( 0) == 0xb4ff5d3e);
```

Comment:
This test is not trivial. Can you please add a trivial test, i.e. which may be explained in comments?

Reply:
What do you mean of trivial test?

Reply:
I can explain this test

Reply:
You can count it by yourself without calculator

Reply:
I mean it is not obvious that result should be `0xb4ff5d3e` and not `0xb4ff5de3`, for instance. Remember, there will be people who'll try to read and understand this code later.
But can we also have anything like `shfl 0x1000, 0x3`?  It is OK to keep your test too.

Reply:
I'm not that smart, even with calculator.
I want to quickly understand what this instruction does. Look at PACK test, for instance:

https://github.com/MIPT-ILab/mipt-mips/blob/4ac60d0ef5d2ccf251d56db860ee4fc11af28014/simulator/risc_v/t/unit_test.cpp#L283-L291